_saas-integrations/google-sheets/v1/google-sheets-v1.md

```diff
@@ -0,0 +1,134 @@
+
+this-version: "1.0"
+
+api: |
+  [Google Sheets v4 AP1](https://developers.google.com/sheets/api){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Free"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+  Stitch's {{ integration.display_name }} integration will generate tables containing data related to metadata and the individual sheets within a spreadsheet. Currently, the Stitch {{ integration.display_name }} integration replicates one spreadsheet at a time. To replicate another spreadsheet, you will need to create another {{ integration.display_name }} integration.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+setup-steps:
+  - title: "Obtain your Spreadsheet ID"
+    anchor: "obtain-spreadsheet-id"
+    content: |
+      1. Go to [{{ integration.display_name }}](http://sheets.google.com){:target="new"} and log into the Google account associated with the spreadsheet you are looking to integrate.
+      2. Open the spreadsheet that you want to use in the integration.
+      3. Your Spreadsheet ID is within the URL to the webpage. In the image below, the portion of the URL within the blue box is the Spreadsheet ID. Keep this readily available to continue with the integration.
+      {% include layout/image.html file="/integrations/google-sheets-spreadsheet-id.png" alt="Google Sheets URL containing the Spreadsheet ID." enlarge=true max-width="850" %}
+  - title: "add integration"
+    content: |
+      4. In the `Spreadsheet ID` field, enter your Spreadsheet ID you obtained from the [previous step](#obtain-spreadsheet-id). Please note that to integrate another spreadsheet, you'll need to repeat these steps over again with another {{ integration.display_name }} integration.
+  - title: "historical sync"
+  - title: "replication frequency"
+  - title: "track data"
+
+
+# -------------------------- #
+#     Replication Details    #
+# -------------------------- #
+
```

Comment:
This needs a key - it's not showing up currently because there isn't a key (list name) for Jekyll to look at. For this section, it should look like this:

```
replication-sections:
  - title: ".."
    anchor: ".."
    content: ".."
```